includes/SpecialGenerateRandomHash.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Miraheze\RemovePII;
+
+use ConfigFactory;
+use FormSpecialPage;
+use Html;
+
+class SpecialGenerateRandomHash extends FormSpecialPage {
+	/** @var ConfigFactory */
+	private $config;
+
+	/**
+	 * @param ConfigFactory $config
+	 */
+	public function __construct( ConfigFactory $config ) {
```

Comment:
Consider naming the ConfigFactory variable to something else than `$config` so that `$config` and `$this->config` are not totally different to reduce potential confusion

Reply:
Done. Thanks!